webpack/assets/javascripts/react_app/common/i18nProviderWrapperFactory.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { IntlProvider } from 'react-intl';
```

Comment:
does this include the functionality of Provider from 'react-redux' ? AFAIU we need it when store is not passed explicitly ?

Reply:
AFAIU the `components/wrapperFactory.js` will wrap it with the react-redux Provider so we should be safe.

This might make it simpler: https://github.com/ratson/react-intl-redux
We can migrate later if we decide to.